test/action_controller/json_api/linked_test.rb

```diff
@@ -137,6 +142,18 @@ def test_render_resource_with_nested_has_many_include
           assert_equal expected_linked, response['included']
         end
 
+        def test_render_resource_with_include_of_custom_key_by_original
+          get :render_resource_with_include_of_custom_key_by_original
+          response = JSON.parse(@response.body)
+          assert response.key? 'included'
+
+          relationships = response['data']['relationships']
+
+          assert relationships.key? 'reviews'
+          assert relationships.key? 'writer'
```

Comment:
Let's just test for what we are interested in, i.e. `reviews`. The rest should be ensured by other tests.
Also, use `assert_include` as it has better reporting format.